### ISSUE TITLE
scanClassPath param is interpreted inversely

### ISSUE THREAD

**Issue by larskn** [Atmosphere/atmosphere#2238]
In `AtmosphereFramework.lookupDefaultBroadcasterType(String)` the init-param `org.atmosphere.cpr.Broadcaster.scanClassPath` is interpreted inversely. `true` disabled the scanning while `false` enables it.

The log correctly says: "Set org.atmosphere.cpr.Broadcaster.scanClassPath to false to change the behavior"

This is caused by this check where  `!Boolean.parseBoolean(drop)` should just be `Boolean.parseBoolean(drop)`. Also the variable name `drop` is a bit confusing since it does actually represent the opposite of "drop".
```
String drop = servletConfig != null ? servletConfig.getInitParameter(ApplicationConfig.AUTODETECT_BROADCASTER) : null;
    if (drop == null || !Boolean.parseBoolean(drop)) {
        //scan broadcasters
```

**Comment by jfarcand** [comment]
@larskn Pull request welcomed :-)

**Comment by larose** [comment]
Fixed in #2338
